test/plugins.test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+var assert = require('assert'),
+    plugins = require('../lib/plugins'),
+    mockery = require('mockery'),
+    sinon = require('sinon');
+
+describe('plugins', function() {
+    beforeEach(function() {
+        this.foobarPlugin = sinon.spy();
+        this.gemini = sinon.spy();
+
+        mockery.registerMock('gemini-foobar', this.foobarPlugin);
+        mockery.enable({
+            warnOnReplace: false,
+            warnOnUnregistered: false
+        });
+    });
+
+    afterEach(function() {
+        mockery.disable();
+    });
+
+    describe('load', function() {
+        it('should load plugin without prefix', function() {
+            var options = {plugins: {foobar: true}};
+            plugins.load(this.gemini, options);
+
+            assert(this.foobarPlugin.calledWith(this.gemini, {}));
```

Comment:
Can you please use `sinon.assert` and `js-must` in tests? The rest of the tests are using this assertions.